yuyo/modals.py

```diff
@@ -921,24 +920,31 @@ def __init__(self, *, ephemeral_default: bool = False) -> None:
             Whether this executor's responses should default to being ephemeral.
         """
         self._ephemeral_default = ephemeral_default
-        self._rows: list[hikari.impl.ModalActionRowBuilder] = self._all_static_rows.copy()
-        self._tracked_fields: list[_TrackedField] = self._all_static_fields.copy()
+        self._rows: list[hikari.impl.ModalActionRowBuilder] = self._static_rows.copy()
+        # TODO: don't duplicate fields when re-declared
```

Comment:
This todo ties into this comment https://github.com/FasterSpeeding/Yuyo/pull/288/files#r1140945074